homeassistant/components/watchyourlan/config_flow.py

```diff
@@ -0,0 +1,63 @@
+"""Config flow for WatchYourLAN integration."""
+
+import logging
+from typing import Any
+
+from httpx import ConnectError, HTTPStatusError
+import voluptuous as vol
+from watchyourlanclient import WatchYourLANClient
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_SSL, CONF_URL
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+# Schema for the user setup form
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_URL): str,
+        vol.Optional(CONF_SSL, default=False): bool,
```

Comment:
Why do we have this, didn't seem like we use it

Reply:
Good catch. I forgot to pass this through to the WatchYourLAN client.